### ISSUE TITLE
error on julia-0.7 branch when using @query

### ISSUE THREAD

**Issue by pstaabp** [queryverse/Query.jl#193]
On the julia-0.7 branch, I tested the example from the documentations:

```
using Query, DataFrames

df = DataFrame(name=["John", "Sally", "Kirk"], age=[23., 42., 59.], children=[3,5,2])

x = df |> @query(i, begin
            @where i.age>50
            @select {i.name, i.children}
          end) |> DataFrame

println(x)
```

and get the following error:
```
LoadError: 
in expression starting at <macrocall>:0

Stacktrace:
 [1] error() at ./error.jl:42
 [2] @from(::LineNumberNode, ::Module, ::Expr, ::Expr) at /Users/XXXXX/.julia/packages/Query/U5o8d/src/Query.jl:33
```

Just reading the source code, it appears that the second argument is not a block.

**Comment by davidanthoff** [comment]
Yes, that is the one thing that I just can't figure out how to get fixed on 0.7... I've been pulling my hair out about this bug for a while now :) At some point I will figure it out, I hope...

**Comment by davidanthoff** [comment]
This is now fixed on ``master``, thanks to @bramtayl!
